crates/orchestrator/src/client.rs

```diff
@@ -334,6 +334,37 @@ impl OrchestratorClient {
         self.get_config_after_collection().await
     }
 
+    /// Sends my public key to the orchestrator to check whether it's duplicate
+    /// And get the bool to indicate whether I can register with my identity
+    /// Real public key registration is for later `post_and_wait_all_public_keys`
+    /// Return false if the public key has already been registered
```

Comment:
Nit: I find this a bit confusing because, from the function name `check_duplicate`, I would expect it to return true if there's duplication. On the other hand, it also makes sense to return `false` for a `check` function when the check fails.

Would it be better if we rename it to `pass_check_unique_keys`?

Reply:
I changed to 'pass_check_no_dup_keys'